pkgs/applications/misc/toot/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pythonPackages, w3m, file, less }:
+
+pythonPackages.buildPythonApplication rec {
+  version = "0.8.0";
+  name    = "toot-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "ihabunek";
+    repo   = "toot";
+    rev    = "${version}";
+    sha256 = "1y1jz4f53njq94zab0icf7jhd4jp10ywm508l4lw6spb69wr7rdy";
+  };
+
+  propagatedBuildInputs = with pythonPackages;
+    [ requests2 beautifulsoup4 future ];
+
+  meta = with stdenv.lib; {
+    description = "Mastodon CLI interface";
+    homepage    = "https://github.com/ihabunek/toot";
+    license     = licenses.mit;
+    platforms   = platforms.unix;
```

Comment:
no need to set platforms as it is already done correctly by `buildPythonApplication`